source/uwp/Renderer/lib/AdaptiveImageRenderer.cpp

```diff
@@ -48,4 +67,687 @@ namespace AdaptiveNamespace
             jsonObject, elementParserRegistration, actionParserRegistration, adaptiveWarnings, element);
     }
     CATCH_RETURN;
+
+    HRESULT XamlBuilder::BuildImage(_In_ IAdaptiveCardElement* adaptiveCardElement,
+                                    _In_ IAdaptiveRenderContext* renderContext,
+                                    _In_ IAdaptiveRenderArgs* renderArgs,
+                                    _COM_Outptr_ IUIElement** imageControl)
+    {
+        ComPtr<IAdaptiveCardElement> cardElement(adaptiveCardElement);
+        ComPtr<IAdaptiveImage> adaptiveImage;
+        RETURN_IF_FAILED(cardElement.As(&adaptiveImage));
+
+        ComPtr<IAdaptiveHostConfig> hostConfig;
+        RETURN_IF_FAILED(renderContext->get_HostConfig(&hostConfig));
+
+        HSTRING url;
```

Comment:
>HSTRING [](start = 8, length = 7)

HString

Reply:
most of this code is bulk copy/pasted, but the `HSTRING` -> `HString` errors should be all fixed now :)

---
In reply to: [326386971](https://github.com/microsoft/AdaptiveCards/pull/3457#discussion_r326386971) [](ancestors = 326386971)